dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.org.eclipse.microprofile.telemetry-1.0.feature

```diff
@@ -3,7 +3,7 @@ symbolicName=io.openliberty.org.eclipse.microprofile.telemetry-1.0
 visibility=private
 singleton=true
 -features=io.openliberty.mpCompatible-6.0
--bundles=io.openliberty.mpTelemetry.1.0.thirdparty; location:="dev/api/third-party/,lib/"
```

Comment:
I'm not sure why we list `dev/api/third-party/` and `lib/` here.

Does the jar actually end up in both locations?

Reply:
Can fix this as a follow up